pkg/webhooks/pod_admission_webhook.go

```diff
@@ -61,7 +61,7 @@ func (p *podWebhook) ValidateCreate(ctx context.Context, obj runtime.Object) (ad
 		return nil, err
 	}
 	if !leaderScheduled {
-		return nil, fmt.Errorf("leader pod not yet scheduled, not creating follower pod. this is an expected, transient error.")
+		return nil, fmt.Errorf("leader pod not yet scheduled, not creating follower pod. this is an expected, transient error")
```

Comment:
We don't add punctuation in any of the other errors, let's remove the "." at the end to keep it consistent.

Reply:
Thanks for your comment, I'll check again later to see if there are any similar errors

Reply:
There should be no similar errors like this in the project.